frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-camelk.feature

```diff
@@ -9,12 +9,20 @@ Feature: CamelK Connector Event Sources
               And user has created or selected namespace "aut-knative-camel-event-source"
 
 
-        @smoke
+        @smoke @5676
         Scenario: CamelSource display in event sources add page: KC-02-TC01
```

Comment:
not related to this PR but we would need to update this as well as with camel k integration operator we don't see camel source but rather multiple sources of the kind kamelets